packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -647,11 +647,26 @@ export default {
                       `Did you mean to include ${declaredDependencyNode.value} in the array instead?`,
                   });
                 } else {
+                  const suggestedDeps = Array.from(
```

Comment:
We have `suggestedDeps` below which is calculated in a more complex way. Why is the same unnecessary here? Should we try to unify the branches and reuse the existing code path? There's already a concept of `unnecessaryDependencies` below which maybe we can reuse. I'm not convinced the code as it is now is enough.